script.js

```diff
@@ -0,0 +1,137 @@
+const firstArrayCreate = () => {
+  const arrayLenght = +prompt(
+    "Specify the number of positions in the array by entering a number from 1 to 10"
```

Comment:
You asking to elements qty be in range from 1 to 10 but never validate that is true